emails/templates/emails/first_forwarded_email.html

```diff
@@ -0,0 +1,573 @@
+    }
+
+    #relay-email-header {
+      padding: 12px 25px;
+    }
+
+    #relay-email-header,
+    #relay-email-footer {
+      font-family: 'inter', Arial, sans-serif;
+      color: #FFFFFF;
+      font-size: 12px;
+    }
+
+    a.container-link {
+      transition: all 0.2s ease;
+      text-underline-offset: 2px;
+      font-family: 'inter medium', Arial, sans-serif;
+    }
+
+    a.container-link:hover {
+      color: #20123ad5;
+      transition: all 0.2s ease;
+    }
+
+    a.container-link {
+      color: #FFFFFF;
+      font-size: 12px;
+    }
+
+    a.container-link:hover,
+    a.container-link:focus,
+    a.container-link:active {
+      transition: all 0.2s ease;
+    }
+
+    .email-trackers-removed-icon {
+      margin-right: 5px;
+      vertical-align: bottom;
+    }
+
+    /* this deals with long email addresses */
+    .forwarded-from-email {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        max-width: 350px;
+    }
+
+    @media screen and (max-width: 768px) {
+      .forwarded-from-email {
+        width: 250px;
+      }
+
+      .footer-block {
+        display: block;
+        width: 100%;
+        text-align: center;
+      }
+
+      .footer-block .container-link img {
+        height: 45px;
+        width: auto;
+      }
+
+      #relay-email-header {
+        padding: 8px 4px;
+      }
+      
+      #hero td {
+        display: block;
+        padding: 0 0 0 24px !important;
+      }
+      .arrow-img {
+        display: none !important;
+      }
+      #hero-mask-img {
+        text-align: center;
+      }
+      #hero-desc {
+        max-width: 100% !important;
+        margin-bottom: 32px;
+      }
+      
+    }
+
+    @media screen and (max-width: 490px) {
+      #our-promise-section *:not(:last-child) {
+        margin-bottom: 32px;
+      }
+      #what-you-can-do-with-relay {
+        padding: 16px;
+      }
+      #what-you-can-do-with-relay td img {
+        min-width: 20px;
+      }
+      #view-your-dashboard {
+        margin: 32px 0 !important;
+      }
+      .forwarded-from-email {
+        width: 200px;
+      }
+      .header-block-left *,
+      .header-block-right * {
+        font-size: 8px !important;
+      }
+      .email-trackers-removed-icon {
+        width: 10px;
+      }
+      .relay-icon {
+        width: 20px;
+      }
+      .arrow-img-two {
+        width: 102px !important;
+      }
+    }
+
+    @media screen and (max-width: 420px) {
+      .arrow-img-two {
+        width: 82px !important;
+        right: 15px;
+      }
+      
+    }
+  </style>
+</head>
+
+<body id="relay-email" style="padding: 0; margin: 0;">
+  <table style="vertical-align: top;min-width: 320px;margin: 0 auto;width:100%" cellpadding="0" cellspacing="0">
+    <tbody>
+      <tr style="vertical-align: top">
+        <td style="word-break: break-word;border-collapse: collapse ;vertical-align: top">
+          <!-- Header -->

```

Comment:
change (blocking): We should remove the header from this template. When I code the back-end, I can re-use the "wrapped email" template. That will also make sure that the header in this email stays in sync with the wrapped email header.

Reply:
Seems like the styling is a little different for mobile screens with the one from the wrapped template.

Asked UX whether they want the styles updated or if I can go with the one in the wrapped template.

(From the wrapped template, mobile looks like this)
![image](https://github.com/mozilla/fx-private-relay/assets/59676643/a92a83cf-250f-4919-aba0-7b7352daaa6c)

(New design for mobile)
![image](https://github.com/mozilla/fx-private-relay/assets/59676643/7ff26c4d-1d4c-48e8-bc66-25743fd57298)

Reply:
@rafeerahman I think it would be better to implement the new redesign for the wrapper template in another PR so that this one can focus on the new template. Not sure if UX answered yet.

Reply:
# - - [ ] ![Web application framework - Wikipedia.pdf](https://github.com/mozilla/fx-private-relay/files/12858296/Web.application.framework.-.Wikipedia.pdf)

Reply:
👍 @lloan . Remove the wrapper from this new email template branch, and make any changes to the wrapped email template in another PR.